antiplag/tasks.py

```diff
@@ -137,19 +136,20 @@ def compare_documents(
                 similarity = text_comparison(doc.text, user_doc.text)
 
                 result_similarity += similarity["first_to_second"]["similarity"]
-                compared_count += 1
             except:
                 # TODO: Should uncomparable documents be included?
                 similarity = None
 
-            results.append(
-                {
-                    "name": str(user_doc),
-                    "percentage": similarity["first_to_second"]["similarity"],
-                    "intervals": similarity["first_to_second"]["intervals"],
-                    "id": user_doc.id,
-                }
-            )
+            if similarity["first_to_second"]["similarity"] > threshold:
```

Comment:
Similarity will now return a value times 100 while the threshold does not change, can we maybe extract it to env?